unit_tests/schema_tests.json

```diff
@@ -0,0 +1,579 @@
+          "description": "must not be blank"
+        },
+        {
+          "payload": "joe",
+          "expected": true,
+          "description": "valid name"
+        },
+        {
+          "payload": "first name",
+          "expected": false,
+          "description": "names do not have spaces in"
+        },
+        {
+          "payload": "names_can_be_reaaly_reaaly_very__very_extremely_long",
+          "expected": true,
+          "description": "names can be long"
+        }
+      ]
+    },
+    "dob":{
+      "pattern_path": "/definitions/PrivateIndividualName/properties/dob",
+      "payload_tests": [
+        {
+          "payload": "",
+          "expected": false,
+          "description": "must not be blank"
+        },
+        {
+          "payload": "01/02/1975",
+          "expected": true,
+          "description": "valid date"
+        },
+        {
+          "payload": "1/2/1975",
+          "expected": false,
+          "description": "invalid date"
+        }
+      ]
+    },
+    "gender":{
+      "pattern_path": "/definitions/PrivateIndividualName/properties/gender",
+      "payload_tests": [
+        {
+          "payload": "Male",
+          "expected": true,
+          "description": "check 'Male' is ok"
+        },
+        {
+          "payload": "Female",
+          "expected": true,
+          "description": "check 'Female' is ok"
+        },
+        {
+          "payload": "Not Specified",
+          "expected": true,
+          "description": "check 'Not Specified' is ok"
+        },
+        {
+          "payload": "something else",
+          "expected": false,
+          "description": "other values are not OK"
+        },
+        {
+          "payload": "",
+          "expected": false,
+          "description": "blank is not ok"
+        }
+      ]
+    },
+    "phone_number":{
+      "pattern_path": "/definitions/PrivateIndividualName/properties/phone_number",
+      "payload_tests": [
+        {
+          "payload": "",
+          "expected": false,
+          "description": "must not be blank"
+        },
+        {
+          "payload": "07234243401",
+          "expected": true,
+          "description": "valid phone numbers start with 07"
+        },
+        {
+          "payload": "082342323401",
+          "expected": false,
+          "description": "does not start with 07 (but right length)"
+        },
+        {
+          "payload": "072342334",
+          "expected": false,
+          "description": "starts with 07 but too short"
+        },
+        {
+          "payload": "082342323143",
+          "expected": false,
+          "description": "starts with 07 but too long"
+        },
+        {
+          "payload": "0723 4243401",
+          "expected": false,
+          "description": "starts with 07 but has a space"
+        },
+        {
+          "payload": "0723 424340",
+          "expected": false,
+          "description": "starts with 07 but has a space"
+        }
```

Comment:
Have we go a duplicate test here as we are checking that the validation fails in both cases due to the space present.